src/Components/SavedSearchAlert/Components/ConfirmationModalHeader.tsx

```diff
@@ -0,0 +1,36 @@
+import { Text, Flex, Join, Separator, Spacer, Pill } from "@artsy/palette"
+import { useSavedSearchAlertContext } from "Components/SavedSearchAlert/SavedSearchAlertContext"
+import { FC } from "react"
+import { useTranslation } from "react-i18next"
+
+export const ConfirmationModalHeader: FC = () => {
+  const { t } = useTranslation()
+  const { pills } = useSavedSearchAlertContext()
+
+  return (
+    <Flex flexDirection="column" mt={-2}>
```

Comment:
I know that we are against not following palette-driven conventions, but still wanted to ask if this change is OK.

| With mt (as in design specs) | Without mt |
|---|---|
|  ![with_top_margin](https://github.com/artsy/force/assets/3934579/fd4a810e-8963-43f8-8223-0df1a4a463c8) | ![without_top_margin](https://github.com/artsy/force/assets/3934579/e2009e8d-18f1-427b-872b-3f86af89c873) |

Reply:
It appears that there's something else going on. There's only a 20px padding on that title to begin with. So I am still against this change, it just looks like you need to remove a margin elsewhere.

Reply:
I think title has m={2} ([code](https://github.com/artsy/palette/blob/main/packages/palette/src/elements/ModalDialog/ModalDialogContent.tsx#L63)) and header has p={2} ([code](https://github.com/artsy/palette/blob/main/packages/palette/src/elements/ModalDialog/ModalDialogContent.tsx#L83)), which gives 40 in total.

Going to remove `mt={-2}` though, maybe padding issue will be fixed in palette somewhen?

Reply:
Ah, yep. I did not know this `header` prop existed. Definitely incorrect. I will fix today.